provision/acc_provision/flavors.yaml

```diff
@@ -25,42 +25,48 @@ cfFlavorOptions: &anchor
 
 flavors:
   # Upstream Kubernetes
+  kubernetes-1.16:
+    desc: Kubernetes 1.16
+    default_version: 4.2 
+    status: null
+    hidden: false
+    order: 5
   kubernetes-1.15:
     desc: Kubernetes 1.15
     default_version: 4.2
     status: null
     hidden: false
-    order: 5
+    order: 6
   kubernetes-1.14:
     desc: Kubernetes 1.14
     default_version: 4.2
     status: null
-    hidden: false
-    order: 6
+    hidden: true
```

Comment:
We will support last 3 releases like upstream. So lets keep this false.

Reply:
Done!